src/gwt/panmirror/src/editor/src/editor/editor.ts

```diff
@@ -257,10 +257,6 @@ export class Editor {
   private maxContentWidth = 0;
   private minContentPadding = 0;
 
-  // keep track of whether the last transaction was selection-only
```

Comment:
Haven't yet seen the GWT side of this, but the original purpose of this was to leave the selection entirely alone if the user had been editing. This is b/c in this case the selection will _already be_ in the perfect location (b/c we are replaying diffs unrolled to inserts/deletes/etc. into source mode as the user edits). It's possible that we've lost this behavior as a result of this change. Let's discuss before you do anything here though both to explore alternative ways to preserve the behavior as well as to evaluate the priority level of preserving it.